dts/bindings/sensor/st,ism330dhcx-common.yaml

```diff
@@ -38,47 +52,31 @@ properties:
       Specify the default accelerometer output data rate expressed in samples per second (Hz).
       Default is power-up configuration.
 
-        Selection
-          0   Power-Down
-          1   12.5Hz
-          2   26Hz
-          3   52Hz
-          4   104Hz
-          5   208Hz
-          6   416Hz
-          7   833Hz
-          8   1660Hz
-          9   3330Hz
-          10  6660Hz
     enum:
-      - 0
-      - 1
-      - 2
-      - 3
-      - 4
-      - 5
-      - 6
-      - 7
-      - 8
-      - 9
-      - 10
+      - 0 # ISM330DHCX_DT_ODR_OFF
+      - 1 # ISM330DHCX_DT_ODR_12Hz5
+      - 2 # ISM330DHCX_DT_ODR_26H
+      - 3 # ISM330DHCX_DT_ODR_52Hz
+      - 4 # ISM330DHCX_DT_ODR_104Hz
+      - 5 # ISM330DHCX_DT_ODR_208Hz
+      - 6 # ISM330DHCX_DT_ODR_416Hz
+      - 7 # ISM330DHCX_DT_ODR_833Hz
+      - 8 # ISM330DHCX_DT_ODR_1666Hz
+      - 9 # ISM330DHCX_DT_ODR_3332Hz
+      - 10 # ISM330DHCX_DT_ODR_6667Hz
+      - 11 # ISM330DHCX_DT_ODR_1Hz6
```

Comment:
Please fill description field as well. Comments are not rendered in https://docs.zephyrproject.org/latest/build/dts/api/bindings/sensor/st%2Clis2dh12-i2c.html#dtbinding-st-lis2dh12-i2c

Reply:
ok sure, my bad...

Reply:
I'm doing as you are propsing. Nevertheless if I see following doc

```https://docs.zephyrproject.org/latest/build/dts/api/bindings/sensor/st,ism330dhcx-i2c.html```

the gyro-range property should not have the correct description, but it has.
Am I missing something?